modules/updating-an-accelerator-profile.adoc

```diff
@@ -7,25 +7,49 @@
+. From the {productname-short} dashboard, click *Settings* -> *Notebook images*.
 +
-The *CustomResourceDefinitions* page reloads to display the search results.
+The *Notebook image settings* page appears. Previously imported notebook images are displayed. To enable or disable a previously imported notebook image, on the row containing the relevant notebook image, click the toggle in the *Enable* column.
+. Click the action menu (&#8942;) and select *Edit* from the list.
 +
-. Click the `AcceleratorProfile` custom resource definition (CRD).
+The *Edit accelerator profile* dialog opens.
+. In the *Name* field, update the accelerator profile name, if applicable.
+. In the *Identifier* field, update the unique string that identifies the hardware accelerator associated with the accelerator profile, if applicable.
+. Optional: In the *Description* field, update the accelerator profile, if applicable.
+. To enable or disable the accelerator profile immediately after creation, click the toggle in the *Enable* column.
+. Optional: Add a toleration to schedule pods with matching taints, if applicable.
+.. Click *Add toleration*. 
 +
-A details page for the custom resource definition (CRD) opens.
-. Click the *Instances* tab.
-. Click the accelerator profile that you want to update. 
-+ 
-A details page for the accelerator profile opens. 
-. Update the YAML code in the embedded editor.
-. Click *Save*.
+The *Add toleration* dialog appears.
+.. From the *Operator* list, select one of the following options:
+* *Equal* - The *key/value/effect* parameters must match. This is the default.
+* *Exists* - The *key/effect* parameters must match. You must leave a blank value parameter, which matches any.
+.. From the *Effect* list, select one of the following options:
+* *None* 
+* *NoSchedule* - New pods that do not match the taint are not scheduled onto that node. Existing pods on the node remain.
+* *PreferNoSchedule* - New pods that do not match the taint might be scheduled onto that node, but the scheduler tries not to. Existing pods on the node remain.
+* *NoExecute* - New pods that do not match the taint cannot be scheduled onto that node. Existing pods on the node that do not have a matching toleration are removed.
+.. In the *Key* field, enter a toleration key. The key is any string, up to 253 characters. The key must begin with a letter or number, and may contain letters, numbers, hyphens, dots, and underscores.
+.. In the *Value* field, enter a toleration value. The value is any string, up to 63 characters. The value must begin with a letter or number, and may contain letters, numbers, hyphens, dots, and underscores.
+.. In the *Toleration Seconds* section, select one of the following options to specify how long a pod stays bound to a node that has a node condition. 
+** *Forever* - Pods stays permanently bound to nodes. 
+** *Custom value* - Enter a value (in seconds) before pods can remain bound to a nodes.
```

Comment:
Same as previous instances:

- Looks like 'nodes' should be singular here.
- Should be able to remove the parentheses.

Adding the same question I've added where this appears in modules/creating-an-accelerator-profile.adoc:

-'before pods can remain bound to a nodes' - Does the custom value define how long pods will stay bound to a node with a node condition before being released?

Reply:
Changed to

Enter a value in seconds to define how long pods stay bound to a node that has a node condition.